util/chunk/row_container.go

```diff
@@ -478,6 +485,20 @@ func (a *SortAndSpillDiskAction) Action(t *memory.Tracker) {
 			}
 			go a.c.sortAndSpillToDisk()
 		})
+		return
+	}
+
+	a.cond.L.Lock()
+	for atomic.LoadUint32(&a.status) == 1 {
+		a.cond.Wait()
```

Comment:
`select * from t order by a union select * from t1 order by a`

Like this SQL, if two `SortExec`s are running concurrently, `SortExec1` is waiting for this condition while `SortExec2` panics, does this `Wait` last forever?

Reply:
No.  SortExec2 panic will not take affect SortExec1 Spilling.  When the function rowContainer.spillToDisk finished, it will set the condition and broadcast.